File: 백준/Bronze/30999. 민주주의/민주주의.cc
#include <iostream>
#include <string.h>

using namespace std;

int main()
{	
	int a, b, count = 0;
	cin >> a >> b;
	for (int i = 0; i < a; i++) {
		string s;
		cin >> s;
		int aa = 0;
		for (int j = 0; j < b;j++) {
			if (s[j] == 'O') {
				aa++;
			}
		}
		if (aa > b / 2) {
			count += 1;
		}
	}
	cout << count;
	return 0;
}

File: 백준/Bronze/30999. 민주주의/민주주의.cs
using System;

namespace hello {
    class h {
        static void Main (string[] args) {
            string[] s = Console.ReadLine().Split();
            int count = 0;
            for (int i =0;i < int.Parse(s[0]);i++) {
                string ss = Console.ReadLine();
                int countt = 0;
                for (int j=0;j < ss.Length;j++) {
                    if (ss[j] == 'O') countt++;
                    else countt--;
                }
                if (countt >= 0) count++;
            }
            Console.WriteLine(count);
        }
    }
}